### ISSUE TITLE
dataNode disk Writabel: false

### ISSUE THREAD

**Issue by ludurr** [chubaofs/chubaofs#1152]
Hi,
**Describe the bug**
Currently I am evaluating this project for our use, so far it looks great, but i am currently blocked by an issue and need to confirm it whether i am correct. 
I have `dataNode WritableNodes":0` I have used datadisk size 1GB. Assume the issue is low size or ?
Isn't really clear from documentation what size of the datadisk must be. I have used the chubaofs-helm installation. 
**Environment**
test env

**To Reproduce**
install the chubaofs from helm and I have used datadisk size 1GB.

**Expected behavior**
Clear description of what are the minimum sizes for datadisk within documentation.

**Additional context**

curl -v "http://127.0.0.1:17010/cluster/stat"
{"code":0,"msg":"success","data":{"DataNodeStatInfo":{"TotalGB":0,"UsedGB":0,"IncreasedGB":0,"UsedRatio":"0.000"},"MetaNodeStatInfo":{"TotalGB":2,"UsedGB":0,"IncreasedGB":0,"UsedRatio":"0.015"},"ZoneStatInfo":{"default":{"DataNodeStat":{"TotalGB":1,"UsedGB":0,"AvailGB":0,"UsedRatio":0,"TotalNodes":1,"WritableNodes":0},"MetaNodeStat":{"TotalGB":2.65,"UsedGB":0.04,"AvailGB":2.61,"UsedRatio":0.02,"TotalNodes":1,"WritableNodes":1}}}}}

Thanks a lot!

**Comment by chengyu-l** [comment]
@ludurr  From the cluster state above, it is easily found that there is only one `DataNode` node and one `MetaNode` node in your cluster. 

```
ZoneStatInfo.default.DataNodeStat.TotalNodes=1
ZoneStatInfo.default.DataNodeStat.WritableNodes=0
ZoneStatInfo.default.MetaNodeStat.TotalNodes=1
```

Maybe you forget to add labeles for `DataNode` or `MetaNode` nodes. you can execute the following commands to add `DataNode` and `MetaNode` nodes. A cluster must have at least three `DataNode` nodes and three `MetaNode` nodes.

```shell
kubectl label node <nodename> chubaofs-metanode=enabled
kubectl label node <nodename> chubaofs-datanode=enabled
```

And there is another issue what the disk space cannot be set too small, otherwise the `DataNode` is unwriteable. `WritableNodes` is at least 3

**Comment by ludurr** [comment]
Hi, Thanks for the reply. 
##Number of Replicas MetaNode/ DataNode: 
That was my intention to use as small number of nodes as possible, but as you suggested i have add labels to additional w/nodes so i can have 3x metaNodes and 3 dataNodes. Now 2 dataNodes showing status as "IsWritable: true", so it worked out. 

DiskSpace: 
I have to keep at 1GB disk siz, but you suggesting at least have 3GB disks sizes ?

Thank you for tips!

**Comment by chengyu-l** [comment]
> Now 2 dataNodes showing status as "IsWritable: true", so it worked out.

It can work when there are at least 3 dataNodes status as "IsWritable: true"
